CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 Magento Functional Testing Framework Changelog
 ================================================
+3.0.0
+-----
+### Upgrade Instructions
```

Comment:
Possible add a note that this upgrade scripts are meant to be used for Test Modules under source code control. If users have old format test modules under `vendor`, those test modules will get upgraded.

Reply:
We should change this in the `upgrade:tests` command I think. Valuable info, just different place, do you agree?

Reply:
That works too. You want me to make a change and add it in?

Reply:
Nm. Let me add this for `upgrade:tests`